package.json

```diff
@@ -41,15 +45,19 @@
     "postcss-reporter": "^3.0.0",
     "postcss-url": "^5.1.1",
     "prettier": "^1.1.0",
-    "suitcss-utils-align": "^0.2.0",
-    "suitcss-utils-display": "^0.4.0"
+    "stylelint": "^7.10.1",
+    "stylelint-config-standard": "^16.0.0"
   },
   "scripts": {
     "setup": "npm run module && npm run virtualenv && npm install && npm run config",
     "watch": "grunt watch",
     "build": "grunt",
-    "lint": "eslint ./Gruntfile.js ./tests ./grunt-tasks ./webcompat/static/js/lib && grunt travis-lint",
-    "fix": "eslint --fix ./Gruntfile.js ./tests ./grunt-tasks ./webcompat/static/js/lib",
+    "lint": "npm run lint:JS && npm run lint:CSS",
+    "lint:JS": "eslint ./Gruntfile.js ./tests ./grunt-tasks ./webcompat/static/js/lib && grunt travis-lint",
+    "lint:CSS": "stylelint ./webcompat/static/css/development/**/*.css",
+    "fix": "npm run lint:fix:JS && npm run lint:fix:CSS",
+    "lint:fix:JS": "eslint --fix ./Gruntfile.js ./tests ./grunt-tasks ./webcompat/static/js/lib",
+    "lint:fix:CSS": "stylelint ./webcompat/static/css/development/**/*.css --fix",
```

Comment:
If you run `stylelint --fix` in your terminal, you can see the option is not implemented, although it is documented. 

Shall we 
1. keep it and document that you have to fix it manually or 
2. remove it and document the fix is not available?
3. Look into something else that has that functionality? 

What do you think @magsout